pandas/core/computation/pytables.py

```diff
@@ -22,7 +22,7 @@
 import pandas.core.common as com
 from pandas.core.computation import (
     expr,
-    ops,
+    ops as ops_func,
```

Comment:
instead of aliasing this one, can we just rename the variable in
```python
ops = PyTablesExprVisitor.binary_ops + PyTablesExprVisitor.unary_ops + ("=",)
```
? `operations` should be fine

Reply:
Done